plotting/plotter.py

```diff
@@ -244,13 +244,13 @@ def plot(self, fig=None):
             )
             plt.close(fig)
 
-            if self.filetype == 'png':
-                buf.seek(0)
-                im = Image.open(buf)
-                with contextlib.closing(BytesIO()) as buf2:
-                    im.save(buf2, format='PNG', optimize=True)
-                    buf2.seek(0)
-                    return (buf2.getvalue(), self.mime, self.filename)
+            #if self.filetype == 'png':
+            #    buf.seek(0)
+            #    im = Image.open(buf)
+            #    with contextlib.closing(BytesIO()) as buf2:
+            #        im.save(buf2, format='PNG', optimize=True)
+            #        buf2.seek(0)
+            #        return (buf2.getvalue(), self.mime, self.filename)
```

Comment:
You can actually just delete this bit of code. If we need to re-visit it in the future it'll be captured in git history.

Reply:
That's really great. Hope we can proceed with the changes.

Reply:
@CubeCode93 Can you remove the commented out lines of code as per the suggestion from @htmlboss above? Again, we don't need commented out code hanging around because it is available in the git history.

Once you have made those changes, go ahead and push them to your branch and we will merge the pull request.

Reply:
@nsoontie Updated.